_maps/shuttles/shiptest/syndicate_twinkleshine.dmm

```diff
@@ -120,7 +120,7 @@
 /area/ship/engineering/atmospherics)
 "aU" = (
 /turf/closed/wall/r_wall/syndicate/nodiagonal{
-	rad_insulation = 0.3
+	rad_fullblocker = 1
```

Comment:
this works the opposite way you think. Fullblock = 1 means the object allows All Radiation to pass through

Reply:
who the fuck coded it like this

Reply:
"coders"

Reply:
no, no. rad_fullblocker = TRUE allows no radiation (*I* coded this:)). for _rad_insulation_, higher is less insulation. however, rad_fullblocker is not really intended for normal turfs (note the documentation). use rad_insulation = 0 instead.